cpu/stm32f7/include/cpu_conf.h

```diff
@@ -44,6 +48,8 @@ extern "C" {
 #define CPU_IRQ_NUMOF                   (98U)
 #elif defined(CPU_MODEL_STM32F769NI) || defined(CPU_MODEL_STM32F767ZI)
 #define CPU_IRQ_NUMOF                   (110U)
+#elif defined(CPU_MODEL_STM32F722ZE)
+#define CPU_IRQ_NUMOF                   (104U)
```

Comment:
just to make sure: does the `stm32f722` need any adaption to the isr vector definitions, also?